other/web-js/client/src/components/TodoItem.js

```diff
@@ -14,6 +14,7 @@ export function TodoItem({ todo, todoActions }) {
     <ListItem>
       <ListItemIcon>
         <Checkbox
+          data-testid="todo-checkbox"
```

Comment:
should this be bluehawk removed?

Reply:
Nope the tests work on the generated artifact projects too